projects/addon-doc/components/page/page.component.ts

```diff
@@ -35,6 +35,9 @@ export class TuiDocPageComponent {
     @Input()
     deprecated: boolean | '' = false;
 
+    @Input()
+    remoteCodeUrl: string | '' = '';
```

Comment:
Maybe we can detect external urls automatically ?

Something similar to the following lines inside `Svg`:
https://github.com/taiga-family/taiga-ui/blob/13c012ffd477de83d5725598172302698f580be1/projects/core/components/svg/svg.component.ts#L131-L133

Reply:
I don't understand, because I need ignore these inputs
<img width="394" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/80d4519d-6fad-461f-82df-c6fe7b4dbbd1">

Reply:
fixed